src/main/java/com/jmb/springfactory/model/dto/TaskDto.java

```diff
@@ -39,6 +45,8 @@
 
   private String reasonRejection;
 
+  private String blockedReason;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "blockedReason" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)